python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -189,6 +200,26 @@ def test_drop(self):
         self.assertEqual(df.drop(col("name"), col("age")).columns, ["active"])
         self.assertEqual(df.drop(col("name"), col("age"), col("random")).columns, ["active"])
 
+    def test_drop_notexistent_col(self):
```

Comment:
can we also add a scala test for easy debugging?

Reply:
sure